src/Algebra/Construct/Flip/Op.agda

```diff
@@ -53,6 +53,14 @@ module _ (≈ : Rel A ℓ) (∙ : Op₂ A) where
   inverse : Inverse ≈ ε ⁻¹ ∙ → Inverse ≈ ε ⁻¹ (flip ∙)
   inverse inv = Prod.swap inv
 
+  zero : Zero ≈ ε ∙ → Zero ≈ ε (flip ∙)
+  zero zer = Prod.swap zer
+
+  module _ (+ : Op₂ A) where
```

Comment:
I know its not the most efficient, but we don't usually nest anonymous modules like this for readability reasons. They tend to very quickly get out of hand and the code becomes difficult to pass. Instead we just list all the arguments again. It also allows you to rename the `∙` operator to the more standard `*` for rings.